circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsS3IntegrationTest/unpartitioned-single-table-avro-schema.yml

```diff
@@ -0,0 +1,10 @@
+table-replications:
+  - source-table:
+      database-name: ${circus-train-runner.database-name}
+      table-name: ct_table_u
+    replica-table:
+      table-name: ct_table_u_copy
+      table-location: s3a://replica/${circus-train-runner.database-name}/ct_table_u_copy
+security:
+  credential-provider: jceks://file/${config-location}/aws.jceks
+      
```

Comment:
Not sure why GH is moaning about no newline here?

Reply:
I think there was some trailing whitespace.